Makefile.ci

```diff
@@ -271,6 +271,13 @@ $(CI_TARGETS): $(NON_CI_GOALS)
 $(CI_VIRTUAL_TARGETS): $(NON_CI_GOALS)
 endif
 
+clean-ci:
```

Comment:
Can we keep the convention that ci targets start with ci-, so `ci-clean` here?

Reply:
It's not a CI target though

Reply:
Kind of. Why would it be called `clean-ci` otherwise?